ci/e2e-tests/test-run.sh

```diff
@@ -603,6 +612,13 @@ case "$OS" in
         vm_image='Canonical:0001-com-ubuntu-minimal-jammy:minimal-22_04-lts-gen2:latest'
         ;;
 
+    'ubuntu:24.04')
+        # az vm image list --all \
+        #     --publisher 'Canonical' --offer 'ubuntu-24_04-lts' --sku 'minimal' \
+        #     --query "[?publisher == 'Canonical' && offer == 'ubuntu-24_04-lts' && sku == 'minimal'].{ sku: sku, version: version, urn: urn }" --output table
```

Comment:
Please remove unused code.

Reply:
It's not unused code per se, it's a comment that documents how the image (in this case, `Canonical:ubuntu-24_04-lts:minimal:latest`) was derived.